components/scream/src/physics/p3/tests/p3_main_unit_tests.cpp

```diff
@@ -27,7 +27,7 @@ static void run_phys_p3_main_pre_loop()
   // TODO
```

Comment:
You should change the pre_loop and post_loop unit tests names for consistency w/ main_main_loop change, right?